### PR TITLE
chore(deps): Update axios to v1.6.3

### PR DESCRIPTION
**Pull request by sumitsuthar**
## Description

Updated axios to v1.6.3

## How to Test

`npm run test`

### CODE REVIEW

**Review by pratik-k2 — APPROVED** [review]

**Comment by bizob2828** [comment]
@sumitsuthar @pratik-k2 are there plans to release a patch of the security agent to address this? We received a report from the agent about this

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
   },
   "dependencies": {
     "@aws-sdk/client-lambda": "^3.436.0",
-    "axios": "1.6.1",
+    "axios": "1.6.3",
```
Comment (bizob2828) [inline package.json:39]: is there a reason why you are pinning this? If it was just `^1.6.0` the next install would pick up the latest

**Comment by sumitsuthar** [comment]
Release is planned for 5th Jan 2024. We will try to push asap.

**Comment by sumitsuthar** [comment]
We have released a new version of security agent. Drafted following PR
https://github.com/newrelic/node-newrelic/pull/1929

File: package.json (hunk 1 above)
Comment (sumitsuthar) [reply package.json:39]: Fix provided in #156

**Pull request merged**